README.rst

```diff
@@ -175,6 +175,13 @@ In some cases, for example when using `GitLab's NPM package registry <https://do
             RUN npm config set '//gitlab.example.com/api/v4/projects/<your_project_id>/packages/npm/:_authToken' '<your_token>'
             RUN npm install '@edx/frontend-component-header@npm:@foo/<your_frontend_component_header_name>@latest'
 
+Running in production
+---------------------
+
+When running this plugin in production, in addition to the standard `DNS configuration <https://docs.tutor.overhang.io/install.html#configuring-dns-records>`__ it is necessary for the domain name that corresponds to the ``MFE_HOST`` setting to be properly configured at a registrar.  Once the domain can be resolved at, for example, ``apps.your.domain``, you should run::
+
+    tutor config save --set MFE_HOST=apps.your.domain
```

Comment:
I would be wary of adding that line. When following along a readme, many users (including me sometimes) just blindly copy-paste any verbatim instructions. I know, I know... but it's a fact of life, I've seen it happen with my own eyes.

Instead, maybe we can indicate that the MFE will be running at the host indicated by:

    tutor config printvalue MFE_HOST

And users would do well to follow the [instructions on setting up the DNS records](https://docs.tutor.overhang.io/install.html#configuring-dns-records).

Reply:
The link to the DNS records is already there, but I agree printing out the value is a better approach.  Will do.